.github/workflows/test_ert_with_slurm.yml

```diff
@@ -71,6 +71,36 @@ jobs:
       run: |
         set -e
 
-        cd test-data/poly_example
-        echo "QUEUE_SYSTEM SLURM" >> poly.ert
+        cp -a test-data/poly_example poly_slurm
+        pushd poly_slurm
+
+        cat << EOF >> poly.ert
+        NUM_REALIZATIONS 5
+        QUEUE_SYSTEM SLURM
+        EOF
+
+        time ert ensemble_experiment poly.ert
+
+
+    - name: Test Torque driver through slurm
+      run: |
+        set -e
+
+        sudo apt install slurm-wlm-torque
```

Comment:
what's `slurm-wlm-torque`?

Reply:
That is the compatibility layer so that a Slurm cluster can expose a Torque shell api